app/views/how-tos/build-basic-prototype/link-pages-together.html

```diff
@@ -128,5 +130,10 @@ <h2 id="link-{{exampleCheckAnswers.url}}-to-your-{{exampleConfirmation.url}}-pag
     and select <strong>Accept and send registration</strong> to check the button works.
   </p>
 
+  {{ warningCallout({
+    heading: "Other links in your pages",
+    HTML: "<p>Your example pages include back links that go back to the template start page. We are not changing them in this tutorial. If you want, you can try and change the links in the back links by using the <a href='https://service-manual.nhs.uk/design-system/components/back-link'>back link component guidance</a>.</p>"
```

Comment:
To future-proof it, this could say "Your example page back links don't yet go to the correct places."?

Reply:
Yes have changed to 

> The back links in your pages do not go to the correct places yet. If you want, you can try and change the links in the back links by using the <a href='https://service-manual.nhs.uk/design-system/components/back-link'>back link component guidance</a>.